app/app/static/custom/ourscript.js

```diff
@@ -342,3 +342,43 @@ window.onload = async (event) => {
 	rerenderTables();
 	
 };
+
+$('#dateSubmit').on('click', function(e) {
+	const startDate = document.getElementById("startDate").value;
+	const endDate = document.getElementById("endDate").value;
+	let startTime = startDate + " 00:00:00.0"
+	let endTime = endDate + " 23:59:59.9"
```

Comment:
You may want to put a reason/comment as to why you are doing this (I know that you are sending a datetime format instead of just date, but would be good to be explicit that you need it to be of datetime format)

Reply:
Will do